src/bp-activity/bp-activity-notifications.php

```diff
@@ -186,6 +192,28 @@ function bp_activity_at_mention_add_notification( $activity, $subject, $message,
 }
 add_action( 'bp_activity_sent_mention_email', 'bp_activity_at_mention_add_notification', 10, 5 );
 
+/**
+ * Notify a member when someone like their post / activity.
+ *
+ * @since BuddyBoss 1.5.6
+ *
+ * @param object $activity           Activity object.
+ */
+
+function bp_activity_like_add_notification( $activity ) {
+	bp_notifications_add_notification(
```

Comment:
added pre-check here

```suggestion
	if ( empty( $activity->id ) || $activity->user_id === bp_loggedin_user_id() ) {
		return;
	}

	bp_notifications_add_notification(
```